src/rgw/rgw_rados.h

```diff
@@ -749,9 +749,9 @@ class RGWSystemMetaObj {
   int use_default(bool old_format = false);
 
 public:
-  RGWSystemMetaObj() {}
-  RGWSystemMetaObj(const string& _name): name(_name)  {}
-  RGWSystemMetaObj(const string& _id, const string& _name) : id(_id), name(_name) {}
+  RGWSystemMetaObj() : cct(NULL), store(NULL) {}
+  RGWSystemMetaObj(const string& _name): name(_name), cct(NULL), store(NULL)  {}
```

Comment:
yes we could but I would leave this right now. If we change it, then we should change the complete code base and this would be out of scope for this patch series atm.